test/regressions/site/src/tests/Subheader/SimpleSubheader.js

```diff
@@ -0,0 +1,10 @@
+// @flow weak
+
+import React from 'react';
+import Subheader from 'material-ui/Subheader';
+
+export default function SimpleSubheader() {
```

Comment:
@nathanmarks Should we group the tests together to speed them up or do you think that keeping them separate is fine?